source/_integrations/rki_covid.markdown

```diff
@@ -0,0 +1,75 @@
+
+
+
+|Sensor  |Type|Description
+|:-----------|:---|:------------
+|`sensor.NAME_count`| number | indicates the confirmed cases.
+|`sensor.NAME_newCases`| number | indicates the new confirmed cases.
+|`sensor.NAME_deaths`| number | indicates the numbers of confirmed death cases.
+|`sensor.NAME_newDeaths`| number | indicates the numbers of new confirmed death cases.
+|`sensor.NAME_recovered`| number | indicates the numbers of confirmed recovery cases.
+|`sensor.NAME_newRecovered`| number | indicates the numbers of new confirmed recovery cases.
+|`sensor.NAME_casesPer100k`| number | indicates cases per 100k.
+|`sensor.NAME_weekIncidence`| number | indicates the week incidence per 100.000 inhabitants.
+|`sensor.NAME_hospitalizationCasesMerged` | float | Hospitalization cases for people in germany summarized  |
+|`sensor.NAME_hospitalizationIncidenceMerged` | float | Hospitalization incidence for people in germany summarized |
+|`sensor.NAME_hospitalizationCasesBaby` | float | Hospitalization cases for people between 00-04 |
+|`sensor.NAME_hospitalizationIncidenceBaby` | float | Hospitalization incidence for people between 00-04 |
+|`sensor.NAME_hospitalizationCasesChildren` | float | Hospitalization cases for people between 05-14 |
+|`sensor.NAME_hospitalizationIncidenceChildren` | float | Hospitalization incidence for people between 05-14 |
+|`sensor.NAME_hospitalizationCasesTeen` | float | Hospitalization cases for people between 15-34 |
+|`sensor.NAME_hospitalizationIncidenceTeen` | float | Hospitalization incidence for people between 15-34 |
+|`sensor.NAME_hospitalizationCasesGrown` | float | Hospitalization cases for people between 35-59 |
+|`sensor.NAME_hospitalizationIncidenceGrown` | float | Hospitalization incidence for people between 35-59 |
+|`sensor.NAME_hospitalizationCasesSenior` | float | Hospitalization cases for people between 60-79 |
+|`sensor.NAME_hospitalizationIncidenceSenior` | float | Hospitalization incidence for people between 60-79 |
+|`sensor.NAME_hospitalizationCasesOld` | float | Hospitalization cases for people older than 80 |
+|`sensor.NAME_hospitalizationIncidenceOld` | float | Hospitalization incidence for people older than 80 |
+
+## Configuration
+Open the `Configuration` of your Home Assistant instance and select `Integrations`.
+Add a new integration, search and select `rki covid`.
+A dialog appears to select a district or state to monitor.
+After submitting the dialog, the newly added sensor entity is available.
+
+**Hint:** Repeat the process to add multiple districts or states.
+
+Districts can also be configured via `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rki_covid
+    districts:
+     - name: 'SK Augsburg'
+     - name: 'SK Berlin Mitte'
+     - name: 'BL Bayern'
+     - name: 'LK München'
+```
```

Comment:
We don't allow YAML configuration for new integration that connects to devices or services.

Reply:
Thank you, removed.